lib/Fmt.ml

```diff
@@ -280,5 +280,8 @@ let fill_text ?epi text =
                       close_box $ fmt "\n@," $ open_hovbox 0
                   | Some _ when not (String.is_empty curr) -> fmt "@ "
                   | _ -> noop )))
-      $ fmt_if (Char.is_whitespace text.[String.length text - 1]) " "
+      $ fmt_if
```

Comment:
What do you think about creating `String.starts_with_whitespace` and `String.ends_with_whitespace` ? that would replace the `space_i` function in `Fmt_ast` as well. This can go in `Import.String` for example.

Reply:
Good idea, done.